src/collision/narrow_phase.rs

```diff
@@ -689,7 +689,7 @@ impl<'w, 's, C: AnyCollider> NarrowPhase<'w, 's, C> {
 #[cfg(debug_assertions)]
 fn log_overlap_at_spawn(
     collisions: Res<Collisions>,
-    added_bodies: Query<(Ref<RigidBody>, Option<&Name>, &Position)>,
+    added_bodies: Query<(Ref<RigidBody>, Option<&Name>, &Position), Without<Sensor>>,
```

Comment:
Hmm? Wouldn't the `Sensor` usually be part of a `Collider`, which may or may not coincide with being a `RigidBody`?
I guess this change helps with the case where the `RigidBody` is holding both the `Collider` and the `Sensor`, but does not help with a `RigidBody` that is the ancestor of a `Collider` with a `Sensor`.